server/controllers/card.js

```diff
@@ -0,0 +1,40 @@
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+const { v4: uuidv4 } = require("uuid");
+
+exports.createCard = asyncHandler(async (req, res, next) => {
+  const { cardTitle, tagColor, userId, columnId, boardId } = req.body;
```

Comment:
Same idea in board, let's utilize parameters where possible like on the board id and oclumn id 👍🏻

Reply:
Did you understand my comment here @uche-okoroafor 😄

Reply:
Paramters in the URL `/board/:boardId/columns/:columnId` etc. More RESTful

Reply:
Ok, I will make the changes but I thought it will be easy and nice to use the body to handle those values.